battle-service/src/main/java/by/ibondarau/tictactoe/battleservice/dto/MoveDto.java

```diff
@@ -3,10 +3,12 @@
 import lombok.Data;
 import lombok.experimental.Accessors;
 
+import java.util.UUID;
+
 @Data
-@Accessors(fluent = true, chain = true)
+@Accessors(chain = true)
```

Comment:
chain-true could be defined at lombok.config file

Reply:
Done